src/test/java/cz/czechitas/automation/TestRunner.java

```diff
@@ -26,7 +26,7 @@ public TestRunner() {
 
     @BeforeEach
     void setUp() {
-        webDriver.get("https://team8-2022brno.herokuapp.com/");
+        webDriver.get("https://datoj23.herokuapp.com/");
```

Comment:
Tohle vytáhnu jako parametr do konfiguračního souboru ještě. Už se budeme přeplácávat přes více lokalit. Ale teď na rychlo už to před víkendem měnit nebudu